common/src/main/java/org/opensearch/ml/common/utils/StringUtils.java

```diff
@@ -191,4 +191,19 @@ public static boolean isEscapeUsed(String input) {
     public static boolean containsEscapeMethod(String input) {
         return patternExist(input, "String\\s+escape\\s*\\(\\s*(def|String)\\s+.*?\\)\\s*\\{?");
     }
+
+    /**
+     * This method will define if we should print out model id with the error message or not.
+     * @param errorMessage
+     * @param modelId
+     * @param isHidden
+     * @return
+     */
+    public static String getErrorMessage(String errorMessage, String modelId, boolean isHidden) {
```

Comment:
We should have a more general class initiated with `client` and expose method for logging, the method should automatically check if current user is super admin, because most of the cases we're not able to get the isHidden.